packages/clay-shared/src/LinkOrButton.tsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayButton from '@clayui/button';
+import ClayLink from '@clayui/link';
+import React from 'react';
+
+export const LinkOrButton: React.FunctionComponent<any> = ({
```

Comment:
hey @bryceosterhaus i'm creating the component but i haven't used it yet. There is a case that I will always pass the `displayType` to` ClayButton` and sometimes I don't want to pass to `ClayLink`, this was breaking some things. So we have to think of a way to structure this. Or we can really separate the props from each other. Any thoughts?

Reply:
Yeah I can see how the `displayType` may not overlap correctly. One option would be to create a sort of map, or we add `linkDisplayType` and `buttonDisplayType`. I'm inclined to use the two individual props, that way we can have some more flexibility to define how the button and link will look.

Reply:
Hmm yeah, I think we can have two individual props for these cases. Sounds good.